### PR TITLE
Tools: add more Vagrant virtual machines

### PR DESCRIPTION
**Pull request by peterbarker**
Tools: Vagrantfile: add trusty64 Vagrant VM

Tools: remove unused variable definition

Tools: Vagrantfile: add zesty64

### CODE REVIEW

**Comment by peterbarker** [comment]
The zesty64 virtual machine can reproduce https://github.com/ArduPilot/ardupilot/issues/7331

From the root directory of an ArduPilot checkout:
```
vagrant up zesty64
vagrant ssh zesty64 -- -X
# now on VM:
sim_vehicle.py
```

File: Vagrantfile [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
+# cd /vagrant && ./waf configure --board=px4-v2 && ./waf build --target=bin/ardusub
+# cd /vagrant ./waf configure --board=navio2 && ./waf build --target=bin/arduplane
+# cd /vagrant ./Tools/autotest/sim_vehicle.py -v ArduPlane # should test JSBSim
+
 # Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
 VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.box = "ubuntu/zesty32"
```
Comment (OXINARF) [inline Vagrantfile]: Is this line still needed?

Comment (peterbarker) [reply Vagrantfile]: Seemed to break when I removed it :-)

Comment (OXINARF) [reply Vagrantfile]: Interesting. But shouldn't it just be `zesty32` then? Otherwise the script doesn't run, right?

**Comment by davidbuzz** [comment]
LGTM

**Comment by jcs224** [comment]
I've tried this one, without running any extra options. Just `vagrant up`. Works great for me. Disregard #7686

**Comment by rmackay9** [comment]
Merged!  Thanks @jcs224 for testing and @davidbuzz for bumping up the priority on it.

**Pull request merged**